zebrad/tests/acceptance.rs

```diff
@@ -443,14 +453,7 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     output.stdout_contains(expected_output)?;
 
-    // If the test child has a cache or port conflict with another test, or a
-    // running zebrad or zcashd, then it will panic. But the acceptance tests
-    // expect it to run until it is killed.
-    //
-    // If these conflicts cause test failures:
-    //   - run the tests in an isolated environment,
-    //   - run zebrad on a custom cache path and port,
-    //   - run zcashd on a custom port.
+    // [Note on port conflict](#Note on port conflict)
     output.assert_was_killed().expect("Expected zebrad with generated config to succeed. Are there other acceptance test, zebrad, or zcashd processes running?");
```

Comment:
This isn't your bug, but we don't want to replace this error - we just want to wrap it with an extra hint.

We'll need to use eyre::WrapErr to use this function.

```suggestion
    output.assert_was_killed().wrap_err("Expected zebrad with generated config to succeed. Are there other acceptance test, zebrad, or zcashd processes running?")?;
```

Reply:
expect won't replace the error, it adds a top level error and prints the value of the error being unwrapped, so this is fine as is imo.

Reply:
https://github.com/ZcashFoundation/zebra/pull/1000/commits/4e6bca466a78290c4a4d9e2906521cdcaf66ca54